lib/logstash/outputs/kafka.rb

```diff
@@ -262,6 +262,9 @@ def retrying_send(batch)
         rescue org.apache.kafka.common.errors.InterruptException => e
           failures << record
           nil
+        rescue org.apache.kafka.common.errors.RecordTooLargeException => e
```

Comment:
[`KafkaProducer::send`](https://kafka.apache.org/0100/javadoc/org/apache/kafka/clients/producer/KafkaProducer.html#send(org.apache.kafka.clients.producer.ProducerRecord,%20org.apache.kafka.clients.producer.Callback) cannot throw a `RecordTooLargeException`, so this PR does not make sense.

Reply:
I would expect it to be thrown from the `future.get()` call a little down the code. Either way this PR does not help because the same (by default infinite) retry logic is still there; #190 proposes a better solution by only retrying sending events when a `RetriableException` is thrown.